src/faqs.njk

```diff
@@ -0,0 +1,383 @@
+                    You can visit their booths to get to know more about them, and the openings in their organization.
+                <p>
+            </div>
+        </div>
+
+        <div class="pt-4 text-lg w-[90%] md:w-[80%]">
+            <li class="flex">
+                <img src="{{ env.baseUrl }}img/assets/circle-lime.svg" alt="Bullet Icon" class="w-5 h-5 mr-2 mt-1.5">
+                <span class="text-xl md:text-2xl font-semibold">September 15, 2025: Devsprints</span>
+            </li>
+            <p class="pt-2 pl-8">
+                During the Devsprints, people contribute to open-source projects, guided by mentors. 
+                This is a nice way to get started with contributing to Python and other open-source projects.
+            <p>
+        </div>
+    </div>
+</div>
+
+
+<div class="main-container md:pt-20 bg-lavender w-full h-auto px-[6%]">
+    <div class="pb-40 md:pb-60 lg:pb-[20%] w-full text-justify lg:w-[70%]">
+        <div class="font-light text-3xl md:text-5xl md:text-center text-[#000000] text-center lg:text-left">
+            <p class="pt-20 md:pt-4 font-semibold">General FAQ's</p>
+            <div class="pt-4 text-[#000000] lg:pr-0">
+                {% call renderDropdown("Where can I find updates related to PyCon India 2025?") -%}
+                    <div class="pt-4 px-4 text-lg">
+                        <p> 
+                            All announcements related to the conference are posted to the <a class="text-blue-600 no-underline" href="https://mail.python.org/mailman3/lists/inpycon-announce.python.org/" target="_blank">PyCon India Announce mailing list</a>. 
+                            If you would like to volunteer, and contribute to the organization of PyCon India, join the <a class="text-blue-600 no-underline" href="https://mail.python.org/mailman/listinfo/inpycon" target="_blank">InPyCon mailing list</a>. 
+                            We also have a presence on social media. All our announcements are posted on our 
+                            <a class="text-blue-600 no-underline" href="https://x.com/pyconindia" target="_blank">X</a>, 
+                            <a class="text-blue-600 no-underline" href="https://linkedin.com/company/pyconindia" target="_blank">LinkedIn</a>, and 
+                            <a class="text-blue-600 no-underline" href="https://www.facebook.com/PyConIndia/" target="_blank">Facebook</a> pages.
+                        </p>
+                    </div>
+                {%- endcall %}
+            </div>
+            <div class="pt-4 text-[#000000] lg:pr-0">
+                {% call renderDropdown("I want to present at the conference. What should I do?") -%}
+                    <div class="pt-4 px-4 text-lg">
+                        <p> 
+                            The call for Proposals is now open. Please visit the 
```

Comment:
CFP is closed now. Maybe we can add about lightning talks?
CC: @anistark @farhaanbukhsh